DeviceAdministration/Infrastructure/Repository/FilterHelper.cs

```diff
@@ -1,26 +1,26 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq;
-using Microsoft.Azure.Devices.Applications.RemoteMonitoring.Common.DeviceSchema;
 using Microsoft.Azure.Devices.Applications.RemoteMonitoring.Common.Exceptions;
 using Microsoft.Azure.Devices.Applications.RemoteMonitoring.Common.Helpers;
+using Microsoft.Azure.Devices.Applications.RemoteMonitoring.Common.Models;
 using Microsoft.Azure.Devices.Applications.RemoteMonitoring.DeviceAdmin.Infrastructure.Models;
 
 namespace Microsoft.Azure.Devices.Applications.RemoteMonitoring.DeviceAdmin.Infrastructure.Repository
 {
     /// <summary>
     /// Testable logic for filtering devices in DocDB
     /// </summary>
-    internal static class FilterHelper
+    public static class FilterHelper
```

Comment:
keep this internal and add InternalsVisibleTo flag to DeviceAdministration/Infrastructure/Properties/AssemblyInfo.cs